examples/hermes-agent/tester.py

```
@@ -0,0 +1,85 @@
+import sys
+import os
+
+def get_pod_name():
+    try:
+        # Use local kubeconfig file if present
+        cmd = ["kubectl"]
+        if os.path.exists("kubeconfig"):
+            cmd.append("--kubeconfig=kubeconfig")
+        cmd.extend(["get", "pods", "-o", "name"])
+        
+        output = subprocess.check_output(cmd, text=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)